### DISCUSSION TITLE
CLI help to save install

### DISCUSSION THREAD

**Discussion by nedlane** [immich-app/immich#6760] [Q&A]
Hi! I self host Immich and I had the database mounted on a different hard drive than my library folder, and that drive failed spectacularly, leaving me with a clean install of Immich, albeit with my files still intact. I have seen discussion here that the best way to bite the bullet and try to save as many of the photos as possible would be to reupload from the library folder using the CLI, but I am not very familiar with how to go about that. Any help is much appreciated!

**Comment by bo0tzz** [comment]
https://immich.app/docs/features/command-line-interface

**Reply by nedlane** [reply]
Do I need to do anything specific except point Immich cli at the old library folder and upload recursively? If not can you advise specifically which folder contains the original images so I can restore?
